Packages/com.unity.inputsystem/InputSystem/Devices/Gamepad.cs

```diff
@@ -637,28 +663,29 @@ public ButtonControl this[GamepadButton button]
         /// <remarks>
         /// When added, a device is automatically made current (see <see cref="InputDevice.MakeCurrent"/>), so
         /// when connecting a gamepad, it will also become current. After that, it will only become current again
-        /// when input change on non-noisy controls (see <see cref="InputControl.noisy"/>) is received.
+        /// when input change on non-noisy controls (see <see cref="InputControl.noisy"/>) is received. It will also
+        /// be available once <see cref="all"/> is queried.
         ///
         /// For local multiplayer scenarios (or whenever there are multiple gamepads that need to be usable
         /// in a concurrent fashion), it is not recommended to rely on this property. Instead, it is recommended
         /// to use <see cref="PlayerInput"/> or <see cref="Users.InputUser"/>.
         /// </remarks>
-        /// <seealso cref="InputDevice.MakeCurrent"/>
-        /// <seealso cref="all"/>
         public static Gamepad current { get; private set; }
 
         /// <summary>
         /// A list of gamepads currently connected to the system.
         /// </summary>
-        /// <value>All currently connected gamepads.</value>
         /// <remarks>
+        /// Returns all currently connected gamepads.
+        ///
         /// Does not cause GC allocation.
         ///
         /// Do <em>not</em> hold on to the value returned by this getter but rather query it whenever
         /// you need it. Whenever the gamepad setup changes, the value returned by this getter
         /// is invalidated.
+        ///
+        /// Alternately, if you want a single gamepad, you can use <see cref="current"/> for example.
```

Comment:
"if you want a single gamepad" sounds a bit odd. maybe "for querying a single gamepad"?